region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -288,6 +288,35 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [1, 3],
+      "name": "G-Mode Morph Power Bomb",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphPowerBomb"
+      ],
+      "clearsObstacles": ["A", "B"]
+    },
+    {
+      "link": [1, 3],
+      "name": "G-Mode Morph Sidehopper Bomb Kill",
```

Comment:
If we just want the runway, we can get most of it by luring the hoppers to the left with Morph.

Reply:
That's a good point. For this strat I was mostly thinking if the door was grey - which would wake Zebes